markdown/events/2023/bytesize_crisprseq.md

```diff
@@ -23,3 +25,65 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[11:38](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=698)
+How you can use nf-core/crisprseq? Basically, you can use the typical nextflow command, where you provide an input sample sheet, the output directory of the profile that you want to run the pipeline with, and then we also have this one single parameter to provide the aligner, by default we're using minimap but you can also choose between bwa or bowtie2, and the reason why we don't have more parameters is because most of them are provided with the sample sheet because they are dependent on the sample.
+
+[12:23](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=743)
+That's how our sample sheet looks. You have the sample name, fastq_1 and fastq_2. If you have only single-end sequencing data you can only provide fastq_1. Then you provide the reference sequence, here it has been shortened for space issues. This reference is the reference the reads will be aligned to, so it's the region where you directed your cut. You also provide the proto-spacer which is the guide RNA that you used in your experiments to direct the cut. Finally, in case that you performed a homology directed repair experiment, you can also provide the template.
+
+[13:11](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=791)
+That's the structure of the output folder, I won't go to all the directories in detail, but you will find all the outputs of all the tools used for pre-processing like the join parent reads. It's all the quality filtering steps because we remove sequencing adapters, we remove low quality reads and mask low quality bases also, and then you also have the output of the alignment and finally the most important folder, which is this one called `cigar`. It's called like that because we parse the edits using the cigar field from the mapping. In these directories you will find some tables and summary tables of the edits and also plots.
+
+[14:15](https://www.youtube.com/watch?v=x_eFQW0nNvo&t=855)
+This is an example of the output plots. We report data quality, meaning that you will have a percentage of reads that have good quality, also the ones that were aligned against the reference. We also report the number of reads that were wild type or the ones that contained indels and from these indels we also classify by filter of quality and if they are located in the expected pickup on the cut site and if they are above the sequencing error rate or not. Finally there's also classification between insertions and deletions, if they are insertions produced by our template and also if these indels are in frame or out of frame because the ones that will be out of frame are the more probable to disrupt a gene function and produce a gene knockout.
```

Comment:
```suggestion
This is an example of the output plots. We report data quality, meaning that you will have a percentage of reads that have good quality, also the ones that were aligned against the reference. We also report the number of reads that were wild type or the ones that contained indels and from these indels we also classify by filter of quality and if they are located in the expected pick on the cut site and if they are above the sequencing error rate or not. Finally there's also classification between insertions and deletions, if there are insertions produced by our template and also if these indels are in frame or out of frame because the ones that will be out of frame are the more probable to disrupt a gene function and produce a gene knockout.
```